pkg/framework/aries/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+/*
+Package aries provides access to the library consumers to create a framework with options. The framework
```

Comment:
Would be an idea to update this text a bit to align with the README.

> A pluggable dependency framework, where implementors can customize primitives via Service Provider Interfaces (SPIs). We have a "batteries included" model where default primitives are included -- such as a key management system (KMS), crypto, data storage, digital hub integration, etc.

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/1a0fabad4135a7495c46315752ad06c326249a89..6d8821c860b3f4c2f12b4e5048eacf470b90bcd2#diff-d8e0c85fc8df82bb6530d2620c04e3daR8).